controller_report_xls/controllers/xfstyle.py

```diff
@@ -0,0 +1,261 @@
+    "Gainsboro": "#DCDCDC",
+    "GhostWhite": "#F8F8FF",
+    "Gold": "#FFD700",
+    "GoldenRod": "#DAA520",
+    "Gray": "#808080",
+    "Grey": "#808080",
+    "Green": "#008000",
+    "GreenYellow": "#ADFF2F",
+    "HoneyDew": "#F0FFF0",
+    "HotPink": "#FF69B4",
+    "IndianRed": "#CD5C5C",
+    "Indigo": "#4B0082",
+    "Ivory": "#FFFFF0",
+    "Khaki": "#F0E68C",
+    "Lavender": "#E6E6FA",
+    "LavenderBlush": "#FFF0F5",
+    "LawnGreen": "#7CFC00",
+    "LemonChiffon": "#FFFACD",
+    "LightBlue": "#ADD8E6",
+    "LightCoral": "#F08080",
+    "LightCyan": "#E0FFFF",
+    "LightGoldenRodYellow": "#FAFAD2",
+    "LightGray": "#D3D3D3",
+    "LightGrey": "#D3D3D3",
+    "LightGreen": "#90EE90",
+    "LightPink": "#FFB6C1",
+    "LightSalmon": "#FFA07A",
+    "LightSeaGreen": "#20B2AA",
+    "LightSkyBlue": "#87CEFA",
+    "LightSlateGray": "#778899",
+    "LightSlateGrey": "#778899",
+    "LightSteelBlue": "#B0C4DE",
+    "LightYellow": "#FFFFE0",
+    "Lime": "#00FF00",
+    "LimeGreen": "#32CD32",
+    "Linen": "#FAF0E6",
+    "Magenta": "#FF00FF",
+    "Maroon": "#800000",
+    "MediumAquaMarine": "#66CDAA",
+    "MediumBlue": "#0000CD",
+    "MediumOrchid": "#BA55D3",
+    "MediumPurple": "#9370D8",
+    "MediumSeaGreen": "#3CB371",
+    "MediumSlateBlue": "#7B68EE",
+    "MediumSpringGreen": "#00FA9A",
+    "MediumTurquoise": "#48D1CC",
+    "MediumVioletRed": "#C71585",
+    "MidnightBlue": "#191970",
+    "MintCream": "#F5FFFA",
+    "MistyRose": "#FFE4E1",
+    "Moccasin": "#FFE4B5",
+    "NavajoWhite": "#FFDEAD",
+    "Navy": "#000080",
+    "OldLace": "#FDF5E6",
+    "Olive": "#808000",
+    "OliveDrab": "#6B8E23",
+    "Orange": "#FFA500",
+    "OrangeRed": "#FF4500",
+    "Orchid": "#DA70D6",
+    "PaleGoldenRod": "#EEE8AA",
+    "PaleGreen": "#98FB98",
+    "PaleTurquoise": "#AFEEEE",
+    "PaleVioletRed": "#D87093",
+    "PapayaWhip": "#FFEFD5",
+    "PeachPuff": "#FFDAB9",
+    "Peru": "#CD853F",
+    "Pink": "#FFC0CB",
+    "Plum": "#DDA0DD",
+    "PowderBlue": "#B0E0E6",
+    "Purple": "#800080",
+    "Red": "#FF0000",
+    "RosyBrown": "#BC8F8F",
+    "RoyalBlue": "#4169E1",
+    "SaddleBrown": "#8B4513",
+    "Salmon": "#FA8072",
+    "SandyBrown": "#F4A460",
+    "SeaGreen": "#2E8B57",
+    "SeaShell": "#FFF5EE",
+    "Sienna": "#A0522D",
+    "Silver": "#C0C0C0",
+    "SkyBlue": "#87CEEB",
+    "SlateBlue": "#6A5ACD",
+    "SlateGray": "#708090",
+    "SlateGrey": "#708090",
+    "Snow": "#FFFAFA",
+    "SpringGreen": "#00FF7F",
+    "SteelBlue": "#4682B4",
+    "Tan": "#D2B48C",
+    "Teal": "#008080",
+    "Thistle": "#D8BFD8",
+    "Tomato": "#FF6347",
+    "Turquoise": "#40E0D0",
+    "Violet": "#EE82EE",
+    "Wheat": "#F5DEB3",
+    "White": "#FFFFFF",
+    "WhiteSmoke": "#F5F5F5",
+    "Yellow": "#FFFF00",
+    "YellowGreen": "#9ACD32"
+}
+
+# Culled from a table at http://www.mvps.org/dmcritchie/excel/colors.htm
+XLWT_COLORS = [
+    (0, 0, 0), (255, 255, 255), (255, 0, 0), (0, 255, 0), (0, 0, 255),
```

Comment:
I will add it in the next commit, but i am waiting for other comments, to fix all in one...